packages/orchestration/README.md

```diff
@@ -198,6 +227,59 @@ const response = await orchestrationClient.chatCompletion({
 return response.getContent();
 ```
 
+### Retrieving Data from the Response
+
+In addition to `getContent()`, other available convenience methods can retrieve the finish reason and token usage.
+Use `response.rawReason` to access the complete HTTP response from the orchestration service.
+
+#### Finish Reason
+
+The Finish Reason indicates the reason for stopping the chat completion request.
+For example, when output gets filtered based on the configuration, the finish reason is `content_filter`.
+
+```ts
+const finishReason = response.getFinishReason();
+```
+
+#### Token Usage
+
+To retrieve the token usage details of the orchestration request, use the following snippet:
+
+```ts
+const tokenUsage = response.getTokenUsage();
+
+logger.info(
+  `Total tokens consumed by the request: ${tokenUsage.total_tokens}\n` +
+    `Input prompt tokens consumed: ${tokenUsage.prompt_tokens}\n` +
+    `Output text completion tokens consumed: ${tokenUsage.completion_tokens}\n`
+);
+```
+
+Remember to initialize a logger before using it.
```

Comment:
Where would the user look for this information? Maybe use console.log instead?

Reply:
I assumed that readers of the guide would be able to look up information about logging on their own, so I didn’t feel the need to explain every detail, especially around configuring loggers.
Since console.log is often flagged by ESLint in many configurations, I thought using a logger would be a better practice.

WDY think?

Reply:
If the idea is that we are providing code snippets that can actually be copied right away and used, then I would not introduce things for which they user has to install a library and figure out how to make it work. I would not assume they already know and have it configured in their projects. `console.log` might complain, but it works directly.